images/master/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
Added double quotes where appropriate, double quoting ```${binaries}``` actually breaks the for loop.